backend/globaleaks/handlers/admin/user.py

```diff
@@ -102,17 +153,19 @@ def db_delete_user(session, tid, user_session, user_id):
 
 
 @transact
-def create_user(session, tid, user_session, request, language):
+def create_user(session, tid, user_session, request, language, wizard:bool = False):
```

Comment:
to be refactored as for the other related code as it breaks Separation of Concerns (SoC) or Encapsulation. Bottom layers should not be aware of upper layers.)